src/lib/components/table/table-settings/dialog/table-settings-dialog.component.spec.ts

```diff
@@ -65,7 +65,9 @@ describe('TableSettingsDialogComponent', () => {
     it('should render column names in options', () => {
         const options: Array<DebugElement> = fixture.debugElement.queryAll(By.css('mat-list-option'));
         const optionTexts: Array<string> = options.map((option: DebugElement) => option.nativeElement.textContent);
-        component.data.columns.forEach((column: ColumnInterface) => expect(optionTexts).toContain(column.key));
+        component.data.columns.forEach((column: ColumnInterface) =>
+            expect(optionTexts.toString()).toContain(column.key)
```

Comment:
Why does this need to be adjusted?
It is an array of strings an toContain should check whether the given value is included in the respective array.
I mean, it was properly working before and you didn't change anything related to the option texts.